### DISCUSSION TITLE
DataCap supported CeresDB for the first time

### DISCUSSION THREAD

**Discussion by qianmoQ** [CeresDB/ceresdb#834] [Show and tell]
[[Plugin] [HTTP] Support ceresdb](https://github.com/EdurtIO/datacap/pull/318)

Welcome to use it to help us support more integration of CeresDB

**Comment by jiacai2050** [comment]
Thanks, looks like a cool project.

```
ping try.datacap.edurt.io
ping: cannot resolve try.datacap.edurt.io: Unknown host
```

It seems the try page is broken

**Reply by qianmoQ** [reply]
The ping operation is disabled

**Reply by jiacai2050** [reply]
Thanks, the page is working, just a little slow.

**Comment by jiacai2050** [comment]
It seems you are using HTTP API to access CeresDB, we have dedicated Java SDK for this purpose, it's gRPC-based and more efficient, and also IMO can make your implementation simpler.
- https://docs.ceresdb.io/sdk/java.html

**Reply by qianmoQ** [reply]
Due to the conflict with the netty-grpc dependency in the system, no good solution has been found and SDK is not used for the time being

**Reply by jiacai2050** [reply]
Does this mean you have two netty-grpc dependencies in your project, and they aren't compatible?

**Reply by qianmoQ** [reply]
It's like this

**Reply by qianmoQ** [reply]
You can implement a `datacap-jdbc-ceresdb` based on SPI
